lib/docker.js

```diff
@@ -232,6 +241,53 @@ Docker.prototype.listImages = function(opts, callback) {
     args.callback(err, data);
   });
 };
+/**
+ * Creates a new volume
+ * @param {Object}   opts     Create options
+ * @param {Function} callback Callback
+ */
+Docker.prototype.createVolume = function(opts, callback) {
+    var args = util.processArgs(opts, callback);
+    var self = this;
+    var optsf = {
+      path: '/volumes/create?',
+      method: 'POST',
+      options: args.opts,
+      statusCodes: {
+        200: true, // unofficial, but proxies may return it
+        201: true,
+        500: 'server error'
+      }
+    };
+
+    this.modem.dial(optsf, function(err, data) {
+      args.callback(err, self.getVolume(data.Name));
+    });
+};
+
+/**
+ * Lists volumes
+ * @param {Options}   opts     Options (optional)
+ * @param {Function} callback Callback
+ */
+Docker.prototype.listVolumes = function(opts, callback) {
+  var args = util.processArgs(opts, callback);
+
+  var optsf = {
+    path: '/volumes',
```

Comment:
This should have a `?` at the end, since the only valid option is `filters` and it is a GET query parameter like in `/containers/json` and `/images/json`